lib/src/api_resources/application_fee.dart

```diff
@@ -1,23 +1,19 @@
 part of stripe;
 
-
-/**
- * [Application Fees](https://stripe.com/docs/api/curl#application_fees)
- */
+/// [Application Fees](https://stripe.com/docs/api/curl#application_fees)
 class ApplicationFee extends Resource {
-
   String get id => _dataMap['id'];
 
-  String objectName = 'application_fee';
+  final String object = 'application_fee';
 
-  static String _path = 'application_fees';
+  static var _path = 'application_fees';
```

Comment:
Would be good to add a comment on what that can be (since apparently it's not necessarily a `String` anymore)